### ISSUE TITLE
[BUG] DownloadContentFromMessage 404 Error

### ISSUE THREAD

**Issue by ilyaborodin** [adiwajshing/Baileys#1278]
When loading some media, I get a 404 error. Is there a way to update the link?

Message example: 
`{"url":"https://mmg.whatsapp.net/d/f/An-UiF_o0RVWAjWkX-nWKpweF7EDfBaxxvuoVKdtLIvV.enc","mimetype":"application/pdf","fileSha256":"TZnID59IzG1JYF3ihw+rDPNccGnf+O0NdZb33NcjYXk=","fileLength":"125540","mediaKey":"jdRfzzDnMJeYR8H6j2yP+7bAZw9IBopwE3XbXYrtPw4=","fileName":"INSS.pdf","fileEncSha256":"mrv+tQ9fg0OIq2IrxL+U28o8Drn5hnbrpMgFnQESK7c=","mediaKeyTimestamp":"1641670348","contactVcard":false}`

**Comment by FaizBastomi** [comment]
If the message is more than 2 weeks or 1 month old, it's usually been deleted from the WhatsApp server.

**Comment by ilyaborodin** [comment]
> If the message is more than 2 weeks or 1 month old, it's usually been deleted from the WhatsApp server.

I tested now the Baileys and web whatsapp. Some images are indeed not available. But at the same time, there are some that are available in web whatsapp but I can’t download it in the Baileys. Here is an example image sent on February 12th:

`{"url":"https://mmg.whatsapp.net/d/f/AtAG93FyS1gyoCyRQcy_K-57MPEqICpJEpiKpkZz2a5i.enc","mimetype":"image/jpeg","fileSha256":"B/AZGc8T/gJ54Eld/0/pfRJSjCccFdkYyAwd1CJFZ1Y=","fileLength":"182932","height":1080,"width":1080,"mediaKey":"6HUjIJV8k+7DQ8wFNJGfD7go4xNt3qlvpQE4/cGQYkA=","fileEncSha256":"hkXArFPwXw1vUbHdSzL6jtXaTCWs25cpB24ZzHK8Nqg=","directPath":"/v/t62.7118-24/56485891_441761897576329_2158489117646874176_n.enc?ccb=11-4&oh=5558d28475706960484733bdc90cb284&oe=61C8F4F4","mediaKeyTimestamp":"1638097322"}`

When I opened this chat in the application. My message was blurred and there was a download button.
